common/app/views/support/ContentFooterContainersLayout.scala

```diff
@@ -23,7 +23,8 @@ object ContentFooterContainersLayout {
     val htmlBlocks = if (isAdvertisementFeature) {
 
       // majority of footer components we don't want to appear on advertisement feature articles
-      Seq(storyPackagePlaceholder)
+      if (showPaidSeriesContainer.isSwitchedOn) Seq(storyPackagePlaceholder, onwardPlaceholder)
+      else Seq(storyPackagePlaceholder)
```

Comment:
Could reduce duplication of `Seq` initialisation:
```
Seq(storyPackagePlaceholder)  :+
if (shouldPaidSeriesContainer.isSwitchedOn) onwardPlaceholder else None
```
Subjective as to whether that is more readable though...